### DISCUSSION TITLE
If I install it as a standalone, will it not be possible to use it with TGWebui?

### DISCUSSION THREAD

**Discussion by Enchante503** [erew123/alltalk_tts#311] [Q&A]
If I install it as a standalone, will it not be possible to use it with TGWebui?

Conversely, if I install it with TGWebui, is it not possible to run AllTalk on its own?
Also, will the virtual environment be shared with TGWebUI?

I'm using Virtualenv (Pyenv) as the virtual environment.

**Comment by erew123** [comment]
On v1 you either run it as part of TGWUI **OR** as a standalone, which the standalone will not integrate into TGWUI.

On v2 you can run it either as part of TGWUI or as a standalone and then there is an extension you can install into TGWUI. https://github.com/erew123/alltalk_tts/blob/alltalkbeta/system/TGWUI%20Extension/AT%20TGWUI%20Remote%20Instructions.txt

V2 is here https://github.com/erew123/alltalk_tts/tree/alltalkbeta

Thanks
